src/improvements/tasks/types/OPCMTaskBase.sol

```diff
@@ -9,7 +9,7 @@ import {IOPContractsManager} from "lib/optimism/packages/contracts-bedrock/inter
 import {IGnosisSafe} from "@base-contracts/script/universal/IGnosisSafe.sol";
 
 import {AccountAccessParser} from "src/libraries/AccountAccessParser.sol";
-import {Action} from "src/libraries/MultisigTypes.sol";
+import {Action, TaskType} from "src/libraries/MultisigTypes.sol";
```

Comment:
I don't think TaskType is used in this contract

Reply:
same comment as above.